app/helpers/application_helper.rb

```diff
@@ -210,4 +210,13 @@ def short_uuid_with_title(uuid)
   def format_uuid(uuid)
     "#{uuid[0,4]}…#{uuid[-4,4]}"
   end
+
+  def field_errors(obj, field)
+    content_tag :div, :class => "error-field" do
+      obj.errors.full_messages_for(field).each do |msg|
+        concat( content_tag(:span, content_tag(:div,"",class:["icon-error","icon-red"]) + msg) )
```

Comment:
- SPAN is an inline element while DIV is a block element; you may nest DIV>SPAN but shouldn't nest SPAN>DIV.
- We're dealing with a _list of errors_, so maybe a UL>LI would be nice to have?
- We can avoid the `visibility:hidden` by only generating the list where there are errors (no need to hide what's not here in the first place).

For example (I'm using `record` and `attr_name` to be closer to Rails' namings):

```ruby
def field_errors(record, attr_name)
  messages = record.errors.full_messages_for(attr_name)
  return if messages.empty?

  content_tag :ul, class: "field-errors" do
    messages.each do |message|
      concat content_tag :li, message, class: "icon-error icon-red"
    end
  end
end
```

Reply:
@straight-shoota noted in #1597 that we're abusing icomoon's `.icon-` definition, and that maybe it should be:

```ruby
concat content_tag :li do
  concat tag :i, class: "icon-error icon-red"
  concat message
end

Reply:
Have to make this to make it work:

```
concat (content_tag :li do
  concat content_tag :i, "", class: "icon-error icon-red"
  concat message
end)
```

Without the parentheses it doesn't work.

Reply:
Oh, I think it has to do with the precedence of ruby blocks? It translates `concat content_tag :li do;end` to `concat(content_tag :li) do;end`. Oops.